src/screens/Connectors/ConnectorAccountDetails.res

```diff
@@ -91,7 +91,7 @@ let make = (~setCurrentStep, ~setInitialValues, ~initialValues, ~isUpdateFlow, ~
     }
     if (
       connector
-      ->getConnectorNameTypeFromString
+      ->getConnectorNameTypeFromString()
```

Comment:
can this be defined at the start of the function rather than calling the  ```
connector
      ->getConnectorNameTypeFromString()
``` multiple times